operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -423,57 +426,80 @@ new customized image.
 
             ```bash
-            S3_ARTIFACT_PATH="boot-images/${NEW_IMS_IMAGE_ID}"
-            echo "${S3_ARTIFACT_PATH}"
+            XNAME=<node-xname>
+            METAL_SERVER=$(cray bss bootparameters list --hosts "${XNAME}" --format json | jq '.[] |."params"' \
+                | awk -F 'metal.server=' '{print $2}' \
+                | awk -F ' ' '{print $1}')
+            echo "${METAL_SERVER}"
             ```
 
-        1. Set the new `metal.server` value.
+        1. Create updated boot parameters that point to the new artifacts.
 
-            ```bash
-            NEW_METAL_SERVER="s3://${S3_ARTIFACT_PATH}/rootfs"
-            echo "${NEW_METAL_SERVER}"
-            ```
+            1. Set the path to the artifacts in S3.
 
-        1. Determine the modified boot parameters for the node.
+               **NOTE:** This uses the `NEW_IMS_IMAGE_ID` variable set in an earlier step.
 
-            ```bash
-            PARAMS=$(cray bss bootparameters list --hosts "${XNAME}" --format json | jq '.[] |."params"' | \
-                sed "/metal.server/ s|${METAL_SERVER}|${NEW_METAL_SERVER}|" | \
-                sed "s/metal.no-wipe=1/metal.no-wipe=0/" | \
-                tr -d \")
-            echo "${PARAMS}"
-            ```
+                ```bash
+                S3_ARTIFACT_PATH="boot-images/${NEW_IMS_IMAGE_ID}"
+                echo "${S3_ARTIFACT_PATH}"
+                ```
 
-            In the output of the `echo` command, verify that the value of `metal.server` is
-            correctly set to the value of `${NEW_METAL_SERVER}`.
+            1. Set the new `metal.server` value.
 
-    1. Update BSS with the new boot parameters.
+                ```bash
+                NEW_METAL_SERVER="s3://${S3_ARTIFACT_PATH}/rootfs"
+                echo "${NEW_METAL_SERVER}"
+                ```
 
-        ```bash
-        cray bss bootparameters update --hosts "${XNAME}" \
-            --kernel "s3://${S3_ARTIFACT_PATH}/kernel" \
-            --initrd "s3://${S3_ARTIFACT_PATH}/initrd" \
-            --params "${PARAMS}"
-        ```
+            1. Determine the modified boot parameters for the node.
+
+                ```bash
+                PARAMS=$(cray bss bootparameters list --hosts "${XNAME}" --format json | jq '.[] |."params"' | \
+                    sed "/metal.server/ s|${METAL_SERVER}|${NEW_METAL_SERVER}|" | \
+                    sed "s/metal.no-wipe=1/metal.no-wipe=0/" | \
+                    tr -d \")
+                echo "${PARAMS}"
+                ```
+
+                In the output of the `echo` command, verify that the value of `metal.server` is
+                correctly set to the value of `${NEW_METAL_SERVER}`.
+
+        1. Update BSS with the new boot parameters.
+
+            ```bash
+            cray bss bootparameters update --hosts "${XNAME}" \
+                --kernel "s3://${S3_ARTIFACT_PATH}/kernel" \
+                --initrd "s3://${S3_ARTIFACT_PATH}/initrd" \
+                --params "${PARAMS}"
```

Comment:
```suggestion
                --params "${PARAMS}" && echo SUCCESS || echo ERROR
```

Reply:
This comment wasn't addressed in your earlier pass. Not sure if it was overlooked.